proto/desmos/profiles/v1beta1/query.proto

```diff
@@ -44,6 +44,12 @@ service Query {
     option (google.api.http).get =
         "/desmos/relationships/v1beta1/blocks/{user}";
   }
+
+  // ChainLink queries the user links for the user having the given address
+  rpc ChainLink(QueryChainLinkRequest) returns (QueryChainLinkResponse) {
+    option (google.api.http).get =
+        "/desmos/relationships/v1beta1/links/{user}/{chain_id}";
+  }
```

Comment:
Instead of a `ChainLink` that returns a single chain link for a single user, I think we should create instead: 

- `UserChainLinks` that returns all the (paginated) chain links for a user 
- `ChainLinks` that returns the paginated list of all chain links for all users